docs/source/getting-started/docker.rst

```diff
@@ -2,7 +2,9 @@
 Set Up a Spark Cluster with Docker
 ==================================
 
-This page describes how to set up a Spark cluster locally on your machine by using Docker containers. This approach is useful if you do not need a high-level of performance, and want to quickly try out the Migrator without having to set up a real cluster of nodes. It requires Docker and Git.
+This page describes how to set up a Spark cluster locally on your machine by using Docker containers. It requires Docker and Git.
+
+.. note:: This approach is useful if you do not need a high-level of performance, and want to quickly try out the Migrator without having to set up a real cluster of nodes. We recommend using a real cluster for production, though.
```

Comment:
Actually the `.. note::` block fails the build:

~~~ text
Extension error (notfound.extension):
Handler <function validate_configs at 0x7fe200ae8820> for event 'config-inited' threw an exception (exception: 'tuple' object has no attribute 'default')
make: *** [Makefile:88: test] Error 2
~~~

Does anyone know how to fix this?

Reply:
It's already fixed, if you trigger the docs build again should pass withot errors.


Context: https://github.com/scylladb/sphinx-scylladb-theme/pull/1157

Reply:
Thank you!